### ISSUE TITLE
ESLint configuration deprecated by the new ESLint 9 release

### ISSUE THREAD

**Issue by Bricklou** [adonisjs/tooling-config#3]
### Package version

1.3.0

### Describe the bug

With the new major release of ESLint, the config files structure changed completely to move into a new format: 
https://eslint.org/blog/2024/04/eslint-v9.0.0-released/
https://eslint.org/docs/latest/use/migrate-to-9.0.0

It seems that with this new format, old configurations can't work any more and need to be updated to support it.

**Comment by Julien-R44** [comment]
Thanks, will do it soon

**Comment by MickL** [comment]
I just migrated my monorepo (also containing Nuxt) to Eslint flat conigs and have no more eslint config for Adonisjs. Would absolutely appreciate a flat config :)

I am not a VS Code user but according to the Nuxt docs you can enable it with the `eslint.experimental.useFlatConfig` setting

**Comment by Bricklou** [comment]
No, now it is `eslint.useFlatConfig` if it hasn't been already detected.

**Comment by luffynando** [comment]
Vscode is released https://github.com/microsoft/vscode-eslint/issues/1644#issuecomment-2173587025
